### DISCUSSION TITLE
Teambition挂载为本地盘（Win和Linux系统），无法写入文件

### DISCUSSION THREAD

**Discussion by ihanting** [AlistGo/alist#3515] [General]
反复测试，在Win和Linux系统中，通过webdav把Teambition挂载为本地盘，写入文件后，本地盘全部显示日期为1999年（无法修改权限），网盘中无文件（只有空文件夹），不知是不是API权限问题。
曾在2月12日在Linux系统中测试一切正常，可以在本地盘写入文件并能在网盘中显示。
不知有没有办法解决或更新。

**Comment by anwen-anyi** [comment]
将 guest 用户启用，默认是不启用状态，强制跳转到登录页面（3.10.1及以上版本）

**Reply by ihanting** [reply]
未解决，测试在alist前台直接上传也是出错
![QQ截图20230219121010](https://user-images.githubusercontent.com/76680260/219922124-6b934d34-e26e-4546-9af6-e24aa5b37e6f.png)

阿里云盘在alist前台上传正常，Teambition不行

**Reply by ihanting** [reply]
alist2和3.11版本进行的测试

**Reply by anwen-anyi** [reply]
我好像回错人了....哈哈尴尬

**Reply by ihanting** [reply]
(⊙o⊙)…

**Reply by anwen-anyi** [reply]
我回复这个的 应该是https://github.com/alist-org/alist/discussions/3514 不知道怎么回复到你这里了 哈哈

**Reply by ihanting** [reply]
我这个问题也没有办法解决耶，基本可以确定Teambition的权限问题，测试了五六款webdav挂载工具，有些提示无权限，有些可以强行把文件放进去（本地显示有），但是网盘中没有文件，另外通过alist前台上传和离线上传都会失败（有时候显示成功了但网盘中还是无文件）

**Comment by JehovahYF** [comment]
是不是使用量太大了
